src/get-started/java-binding.md

```diff
@@ -222,20 +241,78 @@ reference. Light clients also provide access to information on the
 [transaction][exonum-transaction] execution result
 (which may be either success or failure) to their users.
 
+An implementation of the `Transaction#execute` method must be a pure function,
+i.e. it must produce the same _observable_ result on all the nodes of the system
+for the given transaction. An observable result is the one that affects the
+blockchain state hash:<!-- TODO: link the definition? -->
```

Comment:
Should this TODO item be removed, or can you resolve it? The glossary does contain a definition for the blockchain state.

Reply:
I remember that I was dissatisfied with the glossary entry (it didn't clarify things much), hence left a todo.